app/models/mongo/PensionsCYAModel.scala

```diff
@@ -30,8 +32,26 @@ case class PensionsCYAModel(paymentsIntoPension: PaymentsIntoPensionsViewModel,
                             transfersIntoOverseasPensions: TransfersIntoOverseasPensionsViewModel,
                             shortServiceRefunds: ShortServiceRefundsViewModel) {
 
+  // TODO Change once a new journey is added
+  def hasSessionData(journey: Journey): Boolean =
+    journey match {
+      case PensionsSummary              => true // Pension Summary is a special case
+      case PaymentsIntoPensions         => paymentsIntoPension.nonEmpty
+      case AnnualAllowances             => false
+      case UnauthorisedPayments         => false
+      case OverseasPensionsSummary      => false
```

Comment:
Should OverseasPensionsSummary and IncomeFromPensionsSummary also always be true?

Reply:
it's temporarily false, we'll enable one by one when we finish them. Right now we have just one journey

Reply:
done